File: iniciante/c99/1008/1008.java
//QUESTÃO 1008 - Salário


import java.io.IOException;
import java.util.Scanner;

public class Main { 
	public static void main(String[] args) throws IOException {
  	        Scanner sc = new Scanner(System.in);
			            
		int idFuncionario = sc.nextInt();
	        int horasTrabalhadas = sc.nextInt();
		double valorHora = sc.nextDouble();
						            
		double salario = horasTrabalhadas * valorHora;
							            
		String saida = String.format("NUMBER = %d\nSALARY = U$ %.2f", idFuncionario, salario);
								            
		System.out.println(saida);
	}
}

File: iniciante/c99/1008/1008.c
#include <stdio.h>

int main() {

	int numFuncionario, numHoras;
	double valorHora, salario;

	scanf("%d", &numFuncionario);
	scanf("%d", &numHoras);
	scanf("%lf", &valorHora);

	salario = numHoras * valorHora;

	printf("NUMBER = %d\n", numFuncionario);
	printf("SALARY = U$ %.2lf\n", salario);

	return 0;

}
